frontend/src/redux/entities/projects/sagas.ts

```diff
@@ -157,6 +166,14 @@ function* fetchProjects(action: ActionType<typeof fetchProjectsRequest>) {
     true,
     null,
   );
+  const displayedMetrics = projects.reduce(
```

Comment:
Is is necessary to have the displayedMetrics for all projects, even though we only show projects one at a time? It looks like the page ID, parameters, etc… are scoped as the `current` page.

I’ll have a deeper look into the Redux structure and try to figure out a solution more in line with the rest of the Redux mechanics.

Reply:
From what I see in the network tab in the console, we don't call `/projects/<id>` every time we switch to another project, so it may be a problem if we don't store all the metrics when we fetch all projects :/

Reply:
OK got it! You could maybe rename `displayedMetrics` into `displayedMetricsByProjectId` so that it becomes obvious what the id in the object stands for.

Reply:
Another solution would be to send the displayed_metrics with the `GET /projects/first` and `GET /projects` and store that info inside each project’s information. That would however require a number of changes in the backend in this PR. @fargito what do you think?

Reply:
I agree, the metrics should be handled at project level